framework/decode/dx12_dump_resources.cpp

```diff
@@ -1595,7 +1595,7 @@ void Dx12DumpResources::WriteBlockEnd()
     // Dominates profiling (2/2):
     const std::string block =
         json_data_.dump(json_options_.format == util::JsonFormat::JSONL ? -1 : util::kJsonIndentWidth);
-    util::platform::FileWriteNoLock(block.data(), sizeof(std::string::value_type), block.length(), json_file_handle_);
+    util::platform::FileWriteNoLock(block.data(), block.length(), json_file_handle_);
```

Comment:
Should this  be `block.length() * sizeof(std::string::value_type)` for wide characters?

Reply:
Latest changes seem to fix this.